controller/job.go

```
@@ -54,3 +54,24 @@ func (jc *JobController) GetJobsFromAPI(
 	json.NewEncoder(w).Encode(response)
 
 }
+
+// GetJobsConcurrently function
+func (jbc *JobController) GetJobsConcurrently(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	typeNumber := vars["type"]
+	if typeNumber == "even" || typeNumber == "odd" {
+		itemsS := r.FormValue("items")
+		itemsPerWorkerS := r.FormValue("items_per_worker")
+		items, _ := strconv.Atoi(r.FormValue("items"))
+		itemsPerWorker, _ := strconv.Atoi(r.FormValue("items_per_worker"))
```

Do not ignore errors. You should handle them properly. What if the value does not exist, or it's not an integer at all?